front_end/src/components/comment_feed/comment.tsx

```diff
@@ -266,7 +252,7 @@ const Comment: FC<CommentProps> = ({
       onClick: () => setIsReportModalOpen(true),
     },
     {
-      hidden: permissions !== CommentPermissions.CURATOR,
+      hidden: !user?.is_staff,
```

Comment:
I'm assuming that this is equivalent to `IsAdminUser` on the backend?

Reply:
Not ideal, but yeah